test/univariate/continuous/triangular.jl

```diff
@@ -0,0 +1,473 @@
+using Distributions
+using Statistics
+using Test
+
+ϵ = .01
+ψ = 1
+
+function convertstr(t::String, x::T) where T<:Real
+    t == "Float16" && return convert(Float16, x)
+    t == "Float32" && return convert(Float32, x)
+    t == "Float64" && return convert(Float64, x)
+    t == "BigFloat" && return convert(BigFloat, x)
+    t == "Rational" && return convert(Rational, x)
+    t == "Int8" && return convert(Int8, x)
+    t == "Int16" && return convert(Int16, x)
+    t == "Int32" && return convert(Int32, x)
+    t == "Int64" && return convert(Int64, x)
+    t == "Int128" && return convert(Int128, x)
+    t == "BigInt" && return convert(BigInt, x)
+    t == "UInt8" && return convert(UInt8, x)
+    t == "UInt16" && return convert(UInt16, x)
+    t == "UInt32" && return convert(UInt32, x)
+    t == "UInt64" && return convert(UInt64, x)
+    t == "UInt128" && return convert(UInt128, x)
+    # t == "Bool" && return convert(Bool, x)
+end
+
+types = ["Float16", "Float32", "Float64", "BigFloat", "Rational"
+        , "Int8", "Int16", "Int32", "Int64", "Int128", "BigInt"
+        , "UInt8", "UInt16", "UInt32", "UInt64", "UInt128"] #Bool
+n = length(types)^2
+typeslist = Array{String}(undef, n, 2)
+j = 1
+for t in eachindex(types)
+    for u in eachindex(types)
+        typeslist[j, 1] = types[t]
+        typeslist[j, 2] = types[u]
+        global j += 1
+    end
+end
```

Comment:
Can we avoid dealing with strings alltogether? It would be much simpler to just loop over an array/tuple of types, such as `Ts = (Int, Float64, Float32, Rational{Int})`.